ampersand-dom-bindings.js

```diff
@@ -46,12 +51,22 @@ function switchHandler(binding, el, value) {
 }
 
 function getSelector(binding) {
-    if (typeof binding.selector === 'string') {
-        return binding.selector;
-    } else if (binding.hook) {
-        return '[data-hook~="' + binding.hook + '"]';
-    } else {
-        return '';
+    if(binding.firstMatchOnly !== undefined && binding.firstMatchOnly === true) {
```

Comment:
I think that's an error, I wanted to check for undefined/null parameter on the function, not sure how it got there. I should change it for:
```
if(binding && binding.firstMatchOnly)
```